tsl/src/process_utility.c

```diff
@@ -44,14 +46,27 @@ tsl_process_altertable_cmd(Hypertable *ht, const AlterTableCmd *cmd)
 }
 
 void
-tsl_process_rename_cmd(Hypertable *ht, const RenameStmt *stmt)
+tsl_process_rename_cmd(Oid relid, Cache *hcache, const RenameStmt *stmt)
 {
 	if (stmt->renameType == OBJECT_COLUMN)
 	{
+		Hypertable *ht = ts_hypertable_cache_get_entry(hcache, relid, CACHE_FLAG_MISSING_OK);
+		Assert(ht);
 		if (TS_HYPERTABLE_HAS_COMPRESSION_TABLE(ht) || TS_HYPERTABLE_HAS_COMPRESSION_ENABLED(ht))
 		{
 			tsl_process_compress_table_rename_column(ht, stmt);
 		}
+		else
+		{
+			ContinuousAgg *cagg = ts_continuous_agg_find_by_relid(relid);
+			if (cagg)
+			{
+				Hypertable *mat_ht =
+					ts_hypertable_cache_get_entry_by_id(hcache, cagg->data.mat_hypertable_id);
+				Assert(mat_ht);
+				cagg_update_view_definition(cagg, mat_ht);
```

Comment:
Shouldn't we do `cagg_update_view_definition()` for compressed hypertables as well?